### ISSUE TITLE
Fine-tuning Blender 90M to increase level of empathy

### ISSUE THREAD

**Issue by anelkhvan** [facebookresearch/ParlAI#3813]
I am trying to fine-tune Blender 90M to increase its level of empathy by multi-tasking on all tasks (BST, ConvAI2, WoW, and ED) and focusing on ED, but I also want to preserve the original quality of dialogue. What should I set the multitask-weights parameter as? In [https://parl.ai/projects/recipes/](url), this parameter is set to be [1, 3, 3, 3], and I was wondering if these values were chosen because they yield the best results? How can I preserve the high quality dialogue ability of Blender 90M and increase its level of empathy _simultaneously_? 

I ran some tests by trying to fine-tune the pre-trained 90M model for 1 hour on P100:
`from parlai.scripts.train_model import TrainModel 
TrainModel.main( 
    multitask_weights = [1, 2, 2, 3], 
    task='blended_skill_talk,wizard_of_wikipedia,convai2:normalized,empathetic_dialogues',  
    model='transformer/generator', 
    model_file='finetuned/model', 
     
    # initialize with a pretrained model 
    init_model='zoo:tutorial_transformer_generator/model', 
    #init_model='finetuned/model', 
     
    # arguments we get from the pretrained model. 
    # Unfortunately, these must be looked up separately for each model. 
    n_heads=16, n_layers=8, n_positions=512, text_truncate=512, 
    label_truncate=128, ffn_size=2048, embedding_size=512, 
    activation='gelu', variant='xlm', 
    dict_lower=True, dict_tokenizer='bpe', 
    dict_file='zoo:tutorial_transformer_generator/model.dict', 
    #dict_file = 'finetuned/model.dict', 
    learn_positional_embeddings=True, 
     
    # some training arguments, specific to this fine-tuning 
    # use a small learning rate with ADAM optimizer 
    lr=1e-5, optimizer='adam', 
    warmup_updates=100, 
    # early stopping on perplexity 
    validation_metric='ppl', 
    # train at most 30 minutes, and validate every 0.25 epochs 
    max_train_time=3600, validation_every_n_epochs=0.25, 
     
    # depend on your gpu. If you have a V100, this is good (this is taken from ParlAI tutorial, they say batch size  12 is good for V100) 
    batchsize=6, fp16=True, fp16_impl='mem_efficient', 
     
    # speeds up validation 
    skip_generation=True, 
     
    # helps us cram more examples into our gpu at a time 
    dynamic_batching='full', 
)`
I noticed that the quality of dialogue decreases (the model becomes less specific in its answers and repeats itself often) when I increase the weight that is given to the ED task. Is there a way I can prevent that?
